### ISSUE TITLE
Bounty does not display information

### ISSUE THREAD

**Issue by adovrodion** [Joystream/pioneer#3615]
https://dao.joystream.org/#/bounty/preview/640
I pointed out that the bounty had no winner and gave the reason (the work was copied). On the bounty page the reason is not displayed.
You'd think oracle just forgot to judge the bounty...

Maybe the page should show a Rationale window like successful bounties?
Example: 
![image](https://user-images.githubusercontent.com/84832436/190929242-539b45a1-48fa-4ba9-8016-7e09a8766466.png)

**Comment by dmtrjsg** [comment]
Yes certainly makes sense @adovrodion.. We are nearing Carthage release where Bounties will be deprecated, so adding this to icebox for the time being..
